pandas/tests/dtypes/test_dtypes.py

```diff
@@ -795,34 +793,38 @@ def test_update_dtype_errors(self, bad_dtype):
             dtype.update_dtype(bad_dtype)
 
 
-@pytest.mark.parametrize(
```

Comment:
These are cosmetic changes, to make future diffs smaller. We'll be registering DatetimeTZDtype (and need to register PeriodDtype as well).